test/runTest.ts

```diff
@@ -9,14 +9,13 @@ import * as path from 'path';
 
 async function main(): Promise<void> {
     try {
-        const vscodeExecutablePath = await downloadAndUnzipVSCode('stable');
+        const vscodeExecutablePath = await downloadAndUnzipVSCode('insiders');
```

Comment:
This is no longer needed.

Reply:
Oh nice, I guess I'll go around and change all the extensions back